### ISSUE TITLE
Import ISATab not working

### ISSUE THREAD

**Issue by mccalluc** [refinery-platform/refinery-platform#2345]
*(Strongly suspect this is the same issue as #2342, but the recipe is so different that it deserves its own bug.)*

* Specific code commit: 09a2aa2e7e739fb286f370258e899dfe27f5d9c5
* Version of the web browser and OS: OSX FF
* Environment where the error occurred (Vagrant VM and site conf mode or AWS instance): Production test

### Steps to reproduce
Please list all the actions and the input data used:

1. On the dashboard, click "Upload"
2. Pick the "ISATab" tab
3. Enter a URL like "https://beta.stemcellcommons.org/media/file_store/10/82/isa_9905_733878.zip"
4. Hit "Start Import"

### Observed behavior
* The screen immediately fills with an error: Easier to read in the console:
```
IntegrityError at /data_set_manager/import/isa-tab-form/
null value in column "task_id" violates not-null constraint
DETAIL:  Failing row contains (4996, null, PROGRESS, gAJ9cQEoVQdjdXJyZW50cQJNFgxVDHBlcmNlbnRfZG9uZXEDVQcxMDAuMDAlVQV0..., 2017-11-09 15:17:23.425036+00, null, f, eJxrYKotZIzgYGBgSM7IzEkpSs0rZIotZC7WAwBWuwcA).


Request Method: POST
Request URL: http://192.168.50.50:8000/data_set_manager/import/isa-tab-form/
...
```

### Expected behavior
* The ISAtab imports successfully.

**Comment by scottx611x** [comment]
Can reproduce this locally w/ 09a2aa2e7e739fb286f370258e899dfe27f5d9c5

**Comment by scottx611x** [comment]
Fixed by #2346
